lib/WUI/link_content/basic_gets.cpp

```diff
@@ -29,7 +29,7 @@ JsonResult get_printer(size_t resume_point, JsonOutput &output) {
     bool ready = true;
     bool busy = false;
 
-    marlin_client_loop();
+    marlin_update_vars(MARLIN_VAR_MSK_TEMP_ALL | MARLIN_VAR_PRNSPEED | MARLIN_VAR_POS_Z | MARLIN_VAR_PRNSPEED | MARLIN_VAR_PRNSTATE);
```

Comment:
you cannot use those enum values directly
use MARLIN_VAR_MSK(MARLIN_VAR_PRNSPEED)

Reply:
it applies to all your changes